### ISSUE TITLE
Filter for extension

### ISSUE THREAD

**Issue by fabianski7** [manuelvargastapia/libgen_mobile_app#10]
PDF files are very common in libgen, but they are not good for reading in digital readers like the Kindle.

Displaying only files of a certain extension (or not including them), such as epub or mobi, is ideal for those looking for a book to read on this device.

**Comment by manuelvargastapia** [comment]
Yep. That's a nice feature to have. However, it may take some work to implement filtering overall.

If you check the [library](https://www.npmjs.com/package/libgen#usage-searching) I'm using as a base for my [API](https://github.com/manuelvargastapia/libgen_api), you'll see that there is no filtering option. You can search in several fields (title, author, even extension IIRC) and sort the results. However, there is no filtering option. Because of this, I haven't implemented filtering yet, but of course, I would like to. I'll keep this issue open for further discussion and possible solutions.

I'll try to find a workaround when I find some time. Feel free to open a PR here or in the API repo.

**Comment by manuelvargastapia** [comment]
In the upcoming version 1.1.0, the Fiction section support searching by file extension (just like the web version).

**Comment by fabianski7** [comment]
great news!

**Comment by AreYouLoco** [comment]
Any info why is it not suppirted in Sci-Tech section?

**Comment by manuelvargastapia** [comment]
> Any info why is it not supported in Sci-Tech section?

As I tried to explain [previously](https://github.com/manuelvargastapia/libgen_mobile_app/issues/10#issuecomment-754955985), the non-official API I used as the backend for the app doesn't currently allow for that kind of filtering. Indeed, even the official web page only supports it in the Fiction section.
